serde_core/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "serde_core"
+version = "1.0.188" # remember to update html_root_url and serde_derive dependency
+authors = ["Erick Tryzelaar <erick.tryzelaar@gmail.com>", "David Tolnay <dtolnay@gmail.com>"]
+build = "build.rs"
+categories = ["encoding", "no-std", "no-std::no-alloc"]
+description = "A generic serialization/deserialization framework"
```

Comment:
Please update this description to concisely convey the intended usage of this crate in contrast to the serde crate.